tests/features/reports.feature

```diff
@@ -0,0 +1,54 @@
+@reports
+Feature: Reports show the right things
+
+Note: For now we'll have to check them visually
```

Comment:
Hmm, I personally may find them a bit hard to describe. Let me check how hard it would be to create those tests so that we don't even have to think about it.

Reply:
I added a couple report tests for passing Scenarios. It just seemed too hard to describe the desired output.

Reply:
Ah, yup, that's what I meant. Sorry that was unclear.